src/components/plots/styling/violin/SelectData.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
```

Comment:
if we need a violin-specific component for selecting data, why is it not added to `PlotStyling.jsx`?

Reply:
yes, that will be better if we can

Reply:
@ivababukova I am not sure it is a change we want to make. On top of the right bar, we add whatever is in `renderExtraPanels`, and then whatever is configured in `<PlotStyling...>`. For the violin plot, our `SelectData` is the second of 3 items in `renderExtraPanels`. 

We could fix this adding `SelectData`(2nd of 3) and a new simple `DataTransformation` component (3rd of 3) part of `<PlotStyling...>`, but I don't think that this would make our code easier to understand/maintain.

Reply:
yup, I think you are right, let's leave it